content/versions-comparison/4.0-news.json

```diff
@@ -1,55 +1,48 @@
 [
   {
-    "title":"Assets structuration with families and attributes 🖼️",
+    "title":"Asset Manager for your digital assets 🖼️",
     "domains": ["Asset","Governance"],
     "editions": ["EE"],
     "since-version": "4.0",
     "link": "/pim/v4/updates/asset-manager.html#more-flexibility-to-manage-your-assets"
   },
   {
-    "title":"Asset data enrichment 🖼️",
```

Comment:
Why did you remove it? It's the main feature of the Asset Manager, being able to enrich asset (which wasn't possible in the PAM before).